pkg/database/authorized_app.go

```diff
@@ -24,6 +24,9 @@ import (
 
 const (
 	apiKeyBytes = 64 // 64 bytes is 86 chararacters in non-padded base64.
+
+	APIUserTypeDevice = 0
```

Comment:
iota?

Reply:
and should this be a custom type?

Reply:
made it a custom type , plumbed all the way through